kbpagesconfig/editor.go

```diff
@@ -0,0 +1,206 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/keybase/client/go/minterm"
+	"github.com/keybase/kbfs/libpages/config"
+	"github.com/sergi/go-diff/diffmatchpatch"
+	"golang.org/x/crypto/bcrypt"
+)
+
+const configFileName = ".kbp_config"
+
+var term *minterm.MinTerm
+
+func init() {
+	var err error
+	if term, err = minterm.New(); err != nil {
+		fmt.Fprintf(os.Stderr, "opening terminal error: %s\n", err)
+		os.Exit(1)
+	}
+}
+
+// not go-routine safe!
+type kbpConfigEditor struct {
+	kbpConfigPath     string
+	kbpConfig         *config.V1
+	originalConfigStr string
+}
+
+func readConfig(from io.ReadCloser) (cfg config.Config, str string, err error) {
+	buf := &bytes.Buffer{}
+	if cfg, err = config.ParseConfig(io.TeeReader(from, buf)); err != nil {
+		return nil, "", err
+	}
+	if err = from.Close(); err != nil {
+		return nil, "", err
+	}
+	return cfg, buf.String(), nil
+}
+
+// initConfig reads in config file and ENV variables if set.
```

Comment:
Wrong function name? Also, what ENV variables?

Reply:
lol it's generated comment from `cobra`. Sorry I missed it.